src/tools/genpolicy/tests/testdata/createContainer/basic/pod.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox
+spec:
+  runtimeClassName: kata-cc
+  containers:
+    - name: first-test-container
+      image: "quay.io/prometheus/busybox:latest"
```

Comment:
minor to keep in mind: if we can point to a microsoft container image source, could be preferrable, depending on use case.

Reply:
Will try to update to point to mcr in a future PR